index.html

```diff
@@ -0,0 +1,179 @@
+  <title>HTML forms</title>
+<link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <div class="blocks">
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST">
+      <fieldset>
+       <legend>Personal information</legend>
+
+       <div class="form-field">
+      <label for="surname">Surname:</label>
+        <input
+        id="surname"
+        type="text"
+        name="surname"
+        required
+        autocomplete="off">
+      </div>
+
+      <div class="form-field">
+        <label for="name">Name:</label>
+        <input
+        id="name"
+        type="text"
+        name="name"
+        required
+        autocomplete="off">
+      </div>
+
+      <div class="form-field">
+        <label for="age">How old are you:</label>
+        <input id="age"
+        type="number"
+        name="age"
+        placeholder="12"
+        min="12"
+        max="100">
+      </div>
+
+      <div class="form-field">
+        <label for="birthday">Full date of birth:</label>
+        <input
+        id="birthday"
+        type="date"
+        name="birthday"
+        min="2019-01-01">
+      </div>
+
+      <div class="form-field">
+        <label for="accept">I accept the term of the agreement</label>
+        <input
+        id="accept" 
+        type="checkbox" 
+        name="accept_the_term"
+        required>
+      </div>
+      </fieldset>
+    </form>
+  </div>
+      
+      
+  <div class="blocks">
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST">
+      <fieldset>
+       <legend>Registration</legend>
+
+      <div class="form-field">
+        <label for="e-mail">E-mail:</label>
+        <input 
+        id="e-mail" 
+        type="email" 
+        name="e-mail" 
+        placeholder="email@example.com" 
+        required>
+      </div>
+
+
+      <div class="form-field">
+        <label for="password">Password:</label>
+        <input
+        id="password"
+        type="password"
+        name="password"
+        required
+        minlength="5"
+        maxlength="30">
+      </div>
+      </fieldset>
+    </form>
+  </div>
+
+  <div class="blocks">
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST">
+      <fieldset>
+       <legend>An interesting fact about you!</legend>
+
+      <div class="form-field">
+        <label for="love_cats">Do you love cats?</label>
+          <input
+          id="love_cats"
+          type="radio"
+          name="love_cats" 
+          value="yes">
+          Yes
+          <input
+          id="love_cats"
+          type="radio"
+          name="love_cats"
+          value="no">
+          No
+      </div>
+
+      <div class="form-field">
+        <label for="favorite_color">What is your favorite color?</label>
+        <input 
+        id="favorite_color" 
+        type="color" 
+        name="favorite_color" 
+        value="#000000">
```

Comment:
```suggestion
        <input 
          id="favorite_color" 
          type="color" 
          name="favorite_color" 
          value="#000000"
        >
```
Форматирую теги вот так